.github/workflows/label-new-issues.yml

```diff
@@ -0,0 +1,33 @@
+# Description: This workflow automatically adds a `pending-triage` label to issues that need attention
+#
+# Triggered by: newly opened, re-opened or transferred issues
+
+name: Label new issues
```

Comment:
Maybe we should give this a more generic name since it labels existing (reopened) issues?

Reply:
Initially I had just `Label issues` but felt that was too generic. Suggestions?

Reply:
Maybe `Add Triage Label`? Unless we think we will update this workflow to add additional labels?

Reply:
We might in the future. Really what we're doing here is labeling issues that need attention from oncall. Perhaps `Mark issues for triage`.

Reply:
Makes sense to me 👍🏼